content/about/can-tor-help-against-viruses/contents.lr

```diff
@@ -0,0 +1,11 @@
+_model: question
+---
+title: Can Tor help you against viruses?
+---
+seo_slug: use-tor-against-viruses
+---
+description:
+Tor is not an Anti-virus/Malware Software.
+It is an anonymity network,and you can use the Tor browser to avoid getting tracked on the internet.
```

Comment:
Tor Browser isn't only about online tracking, but also to bypass online censorship.

Reply:
It's Tor Browser.